tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -1400,4 +1401,81 @@ describe(__filename, () => {
       reply: true,
     });
   });
+
+  it('scrolls to the top of the page when an error is rendered', () => {
+    const _window = {
+      scroll: sinon.spy(),
+    };
+    const root = renderUserProfileEdit({ _window });
+
+    sinon.assert.notCalled(_window.scroll);
+
+    root.setProps({
+      errorHandler: createStubErrorHandler(new Error('some error')),
+    });
+
+    sinon.assert.calledWith(_window.scroll, 0, 0);
+  });
+
+  it('does not scroll if we already scrolled because of an error', () => {
```

Comment:
This test is described as `we already scrolled because of an error` but it doesn't actually cause a scroll because of an error first, it just starts off with an activated errorHandler, so I don't think the test case (exactly) matches the description. Would it be better to do what you've done in `it('scrolls to the top of the page when an error is rendered')`, above, as the set-up for this test, so we actually do scroll first, and then check that we don't scroll again on re-render?

Either that, or change the description if what we're really testing is a pre-existing error, and not whether we've already scrolled or not.

Reply:
ah right, I wanted to avoid the scroll set-up and directly inject the error handler with an error because this was not super-important (since in both cases, it leads to the same result), but I understand that it is not crystal clear, so I have added the set-up to first scroll and then make sure we do not scroll again.